ThirdParty/UIImageColors.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  UIImageColors.swift
+//  https://github.com/jathu/UIImageColors
+//
+//  Created by Jathu Satkunarajah (@jathu) on 2015-06-11 - Toronto
+//  Original Cocoa version by Panic Inc. - Portland
+//
```

Comment:
For any additional third party sources we add, make sure to also update our license page to give credit and display the author's license in Licenses.html